mitodl/ocw-studio
# Added omnibus site config and pared down course config

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #298 

#### What's this PR do?
- Adds a site config that is intended to exercise all of our widget types
- Removes fields that aren't OCW course site-related from the OCW course site config
- Updates the `override_site_config` command to update the omnibus config by default

#### How should this be manually tested?
Run the migrations, and run the `override_site_config` command. You should be able to use the omnibus config for new sites, and that config should exercise every available widget
